runbotics-orchestrator/src/main/docker/postgresql.yml

```diff
@@ -64,6 +64,7 @@ services:
     image: runbotics/runbotics-desktop:latest
     environment:
       - RUNBOTICS_ENTRYPOINT_URL=http://host.docker.internal:3000
+      - RUNBOTICS_SCHEDULER_ENTRYPOINT_URL=http://host.docker.internal:4000
```

Comment:
frontend also redirects `/api` to the backend don't we need that too?

Reply:
we use normal RUNBOTICS_ENTRYPOINT_URL in that case, because we have proxy for /api in next.config.js